website/source/docs/guides/managing-acl-policies.html.md

```diff
@@ -0,0 +1,150 @@
+| `service` and `service_prefix`   | Service rules grant privileges for service-level registration, including adding services to the catalog. 
+| `operator`        | The operator grants privileges for datacenter operations, including interacting with Raft. 
+
+
+On the API pages, each endpoint will have a table that includes required ACLs. The node health endpoint, shown below, requires node and service read to view all checks for the specified node. 
+
+![API Health Endpoint](/static/img/consul/api-endpoint.png 'Screenshot of Health endpoint page')
+
+Finally, before using a token in production, you should test that it has the correct privileges. If the token being used is missing a privilege, then Consul will issue a 403 permission denied error. 
+
+```sh
+$ consul operator raft list-peers
+Error getting peers: Failed to retrieve raft configuration: Unexpected response code: 403 (rpc error making call: Permission denied)
+``` 
+
+### Operations Example: Listing Consul Agents
+
+To view all Consul agents in the datacenter, you can use the `consul members` CLI command. You will need at least read privileges for every agent in the datacenter. Depending on your threat model you could have either of the following policies. 
+
+An individual rule and policy for each agent: this requires you to add a new policy to the token every time a new agent is added to the datacenter. Your token would have as many policies as agents in the datacenter. This method is ideal for a static environment, and for maximum security.
+
+```hcl
+agent "server-one" 
+  { policy = "read"
+  }
+```
+
+If you have a dynamic or very large environment, you may want to consider creating one policy that can apply to all agents, to reduce the Operation Team’s workload. 
+
+```hcl
+agent_prefix "" 
+  { policy = "read"
+  }
+```
+
+## Secure Access Control: Operator-Only Access
+
+The most secure access control implementation restricts tokens with `acl=”write”` policies to only one or a few trusted operators. Tokens with the policy `acl = "write"` grant the holder unlimited privileges, because they can generate tokens with any other resource and policy. The operators are responsible for creating all other policies and tokens that grant limited access to the datacenter. We refer to this implementation as the operator-only implementation. This implementation type is the most secure, and most complex to manage.  
+
+For this implementation type operators are responsible for managing policies and tokens for: 
+- service registration
+- Connect proxy registration
+- intention management
+- agent management
+- API, CLI, and UI access
+
+For static or non-containerized workflows, this implementation type is straightforward and the operator's workload scales linearly. If the workflow is dynamic, implementing an automation tool will ensure the operator's workload does not scale exponentially. 
```

Comment:
🤔 static vs dynamic doesn't really effect linear vs exponential scaling - in both cases the total work to be done in linear in the number of instances provisioned. It's more that in a dynamically scheduled environment, automation is _necessary_ to avoid having and manual operator step in deploying a new service. Even then it might be OK if you just distribute the same token to all instances (common although not ideal) - then the operator only is needed when a brand new service spins up.

I'd suggest maybe rewording this so we don't conflate scale of tokens needed with static vs dynamic requirement to provision new services/tokens?